pages/wiki/creating-an-asteroid-app.md

```diff
@@ -152,10 +151,88 @@ Change the following `Environment` variables:
 
 Your app should now be able to run from the application when you click the start button in the bottom left sidebar.
 
+# Learn QML
+
+Main docs: https://doc.qt.io/ (AOS uses qt5)
+
+AOS specific component/API: https://github.com/AsteroidOS/qml-asteroid
+
+Examples of use of most of those can be found by searching the AsteroidOS codebase: https://github.com/search?q=org%3AAsteroidOS+StatusPage&type=code
```

Comment:
Let's join this with the above paragraph since it's about qml-asteroid.
Also a grammar nit "Example uses of qml-asteroid components can be..."